UmbracoFood/Controllers/Api/OrderApiController.cs

```diff
@@ -99,7 +99,34 @@ public void PutEditOrder(EditOrderViewModel model)
             {
                 orderService.ChangeStatus(model.OrderId, (OrderStatus) model.Status);
             }
+        }
+
+        [HttpDelete]
+        public void DeleteOrder(int id)
+        {
+            if (id < 1)
+            {
+                throw new Exception("ID nie może być niższe niż 1");
+            }
+
+            var order = orderService.GetOrder(id);
+            if (order == null)
+            {
+                throw new Exception("Zamówienie nie istnieje");
+            }
+
+            orderService.RemoveOrder(id);
```

Comment:
Ale metoda Delete/Remove działa raczej inaczej. Tam usuwasz rekordy, które spełniają dane kryteria. W tym przypadku muszą mieć dane ID. Jeśli nie będzie rekordu o danym ID, to po prostu nic nie usunie, i nie ma wyjątku.